lusSTR/scripts/marker.py

```diff
@@ -69,12 +69,17 @@ def _uas_bases_to_trim(self):
         function determines the number of bases that need to be trimmed from the full amplicon
         sequence to recover the UAS core sequence.
         """
-        if self.uas:
+        if self.software == "uas":
             return 0, 0
         elif self.kit == "forenseq":
             return self.data["Foren_5"], self.data["Foren_3"]
         elif self.kit == "powerseq":
-            return self.data["Power_5"], self.data["Power_3"]
+            if self.locus == "D16S539" and self.software == "genemarker":
+                return self.data["Power_5"], (self.data["Power_3"] - 3)
+            elif self.locus == "D8S1179" and self.software == "genemarker":
+                return (self.data["Power_5"] - 5), (self.data["Power_3"] - 5)
+            else:
+                return self.data["Power_5"], self.data["Power_3"]
```

Comment:
Using the GeneMarker software, only two loci (D16 and D8) produced different sequences (STRait Razor contains a few additional bases). This accounts for these differences.